sc-platform-api/src/api/article/article.controller.ts

```diff
@@ -7,6 +7,7 @@ import { Article } from '../../database/entities/article.entity';
 import { AccessGuard } from '../../security/jwt/access/access.guard';
 import { ArticleDocumentation } from '../../documentation/article';
 import { MapInterceptor } from '@automapper/nestjs';
+import { ArticleByIdPipe } from './article.pipe';
 
 @Controller('article')
```

Comment:
Change controller path to `articles`
```suggestion
@Controller('articles')
```